_posts/2021-06-09-may-rubygems-updates.md

```diff
@@ -0,0 +1,48 @@
+---
+title: May 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in May.
+
+## RubyGems News
+
+This month in RubyGems, we released new versions for Bundler [`2.2.18`, `2.2.19`](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2218-may-25-2021) and RubyGems [`3.2.18`, `3.2.19`](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3218--2021-05-25) and focused on shipping a definitive fix for the dependency confusion issues that have been affecting Bundler for years. We finally managed to provide a fix ([#4609](https://github.com/rubygems/rubygems/pull/4609)) with `bundler 2.2.18`.
+
+In addition to that, RubyGems saw several bug fixes and updates this month, some of which include the following:
+
+- fixed a resolution issue where gems were being unintentionally removed from the lockfile - [#4580](https://github.com/rubygems/rubygems/pull/4580).
+- shipped a fix in RubyGems to improve the reproducibility of building packages - [#4610](https://github.com/rubygems/rubygems/pull/4610).
+- shipped other minor improvements, and some internal changes to our development environment like moving away from `minitest` in favor of `test-unit`.
+
+Checkout [RubyGems](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3218--2021-05-25) and [Bundler](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2218-may-25-2021) for the full changelog of the new versions shipped this month! 
+
+In May, Rubygems gained [132 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-05-01%7D...master@%7B2021-05-31%7D), contributed by 10 authors. There were 2419 additions and 2118 deletions across 228 files.
+
+## RubyGems.org News
+
+In May, RubyGems.org saw several bug fixes and updates, some of which include the following:
+
+- investigated and fixed cache poisoning by using `x-forwarded-scheme` header gotten from a [HackerOne Report](https://hackerone.com/bugs?subject=rubygems&report_id=1184815).
```

Comment:
```suggestion
- investigated and fixed cache poisoning by using `x-forwarded-scheme` header. The issue was reported on HackerOne.
```
perhaps we don't need to add the link since it is private.